packages/shared-ux/code_editor/impl/code_editor.tsx

```diff
@@ -380,6 +392,22 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
         if (codeActions) {
           monaco.languages.registerCodeActionProvider(languageId, codeActions);
         }
+
+        if (true) {
```

Comment:
Ignore this if true, but basically debugged that `monaco.languages.registerInlayHintsProvider(languageId, inlayHintsProvider);` is correctly called